dev-libs/libbase58/libbase58-0.1.4.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A library for encoding/decoding base58"
+DESCRIPTION="libbase58 provides encoding and decoding for the base58 format, \
+used in Bitcoin addresses."
+HOMEPAGE="https://github.com/bitcoin/libbase58"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libbase58/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="97e8c3387fd4ed90208a8101e658743efbb6e21714e594f0828e6414f92c0b1d"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [[ "$targetArchitecture" == "x86_gcc2" && -z "$secondaryArchSuffix" ]]; then
```

Comment:
I do agree that
```
if [[ "$targetArchitecture" == "x86_gcc2" && -z "$secondaryArchSuffix" ]]; then
```
works fine, but it would be better to replace it by:
```
if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
```
There are 4 reasons for this:
1) we avoid a bash only syntax (the "[[ ... ]]" syntax and the "==" are bash-specific).
2) it would also work (without any changes) for x86_gcc2 *secondary* arch, should we ever decide to add x86_gcc2 to SECONDARY_ARCHITECTURES.
3) it makes it clear that the patch is only required when building for x86_gcc2 (primary or secondary).
4) we can have the same result with a single test, instead of a double test.